test/worker.ts

```diff
@@ -23,13 +23,15 @@ Comlink.expose({
       const callback = makeRunnerCallback(extras, {
         input: inputCallback,
         output: outputCallback,
+        other: (type, data) => type + "-" + JSON.stringify(data),
       });
       const pyodide = await pyodidePromise;
       if (extras.interruptBuffer) {
         pyodide.setInterruptBuffer(extras.interruptBuffer);
       }
       const runner = pyodide.pyimport("python_runner").PyodideRunner();
       runner.set_callback(callback);
+      pyodide.pyimport("builtins").runner = runner;
```

Comment:
This is to make running tests more compact, I presume?

Reply:
It was the easiest way to make `runner` available in Python code to test `runner.callback(...)` without having to write a Python file which does things like subclass PyodideRunner.